uni/lib/controller/local_storage/preferences_controller.dart

```diff
@@ -281,11 +282,7 @@ class PreferencesController {
   /// Decrypts [base64Text].
   static String? decode(String base64Text) {
     final encrypter = _createEncrypter();
-    try {
-      return encrypter.decrypt64(base64Text, iv: iv);
-    } catch (_) {
-      return null;
```

Comment:
Why was this removed?

Reply:
I think is to sentry catch the error

Reply:
Why is this a relevant error for sentry to catch? This happens if the password was encrypted with a different key/IV - we changed it ourselves - or if the store gets miraculously corrupted - unlikely. But in either case, we just want the user to perform login again, no?
Either way, if we wanted the exception to go to sentry, wouldn't it be better to add the code in the catch block to report the exception and return null either way?